packages/vscode-plugin/.vscode/settings.json

```diff
@@ -1,13 +1,6 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-	"files.exclude": {
-		"out": false, // set this to true to hide the "out" folder with the compiled JS files
-		"dist": false // set this to true to hide the "dist" folder with the compiled JS files
-	},
-	"search.exclude": {
-		"out": true, // set this to false to include "out" folder in search results
-		"dist": true // set this to false to include "dist" folder in search results
-	},
+	"search.exclude": { "build": true },
```

Comment:
I'm not familiar with VSCode settings like this. I'm going assume they make sense. Maybe someone like @lukasmwerner has an opinion?

Reply:
Yeah these all look reasonable to me.